packages/gatsby/src/schema/infer/merge-inferred-composer.js

```diff
@@ -0,0 +1,165 @@
+const _ = require(`lodash`)
```

Comment:
it would be handy if you write some jsdoc comments above each function export to explain what it does as it's hard to follow if you have almost no knowledge of it.

Reply:
:+1: good idea